client/utils.lua

```diff
@@ -64,10 +66,85 @@ if GetConvarInt('ox_target:drawSprite', 1) == 1 then
             end
         end
 
+
+
         return n > 0 and inRange, newZone
     end
 
+
+    function utils.getNearbyTarget(coords)
+        coords = coords or GetEntityCoords(PlayerPedId())
```

Comment:
Coords is never defined, except we already have it elsewhere in the code and could easily call it.
We also don't use PlayerPedId anywhere in this resource, yet suddenly!